### PR TITLE
Fix: Nft section zeroed id

### LINKED ISSUE #1307: [Task]: Fix wrong NFT ID
### Task description

`/alphanet/addr/rms1qpj0hznjylfnfjayycch26tk3pr4u6vmsmcjrqpyzsdxajmuy5a0g03mukz?tab=Nfts` Shows `0x000..` ID for the NFT 
image

### PR DESCRIPTION
**Pull request by brancoder**
# Description of change

closes #1307 

## Type of change

- Bug fix (a non-breaking change which fixes an issue)

### CODE REVIEW

**Review by VmMad — APPROVED** [review]
🚀

File: client/src/app/components/nova/address/section/nft/Nft.tsx [hunk 1]
```diff
@@ -56,6 +65,10 @@ const Nft: React.FC<NftProps> = ({ nftOutput }) => {
             if (metadata) {
                 setIssuerId(nftIssuerId);
             }
+            const nftId = nftOutput.nftId === ZEROED_ID ? Utils.computeNftId(outputId) : nftOutput.nftId;
```
Comment (begonaalvarezd) [inline client/src/app/components/nova/address/section/nft/Nft.tsx:68]: Could we maybe use the same approach as we have in stardust just so the code is similar and we can maintain both networks with the same logic? `const nftId = HexHelper.toBigInt256(nftOutput.nftId).eq(bigInt.zero) ? Utils.computeNftId(outputId) : nftOutput.nftId;`

File: client/src/app/components/nova/address/section/nft/Nft.tsx [hunk 2]
```diff
@@ -56,6 +63,10 @@ const Nft: React.FC<NftProps> = ({ nftOutput }) => {
             if (metadata) {
```
Comment (msarcev) [inline client/src/app/components/nova/address/section/nft/Nft.tsx:63]: Will this hook "recompute" after `metadata` is set (on L60) ? I would say maybe not as metadata is not in the dependencies 🤔

Comment (msarcev) [reply client/src/app/components/nova/address/section/nft/Nft.tsx:63]: Ah, i think it was supposed to be `if (nftIssuerId) {...`

**Pull request merged**